test/ts/index.tsx

```diff
@@ -1,38 +1,49 @@
-import { h, app, ActionsType, View } from "hyperapp"
+import { h, app, View, StateType, UnwiredActions, UnwiredAction } from 'hyperapp'
 
 namespace Counter {
-  export interface State {
+  export interface State extends StateType {
     count: number
+    clickCount: number
```

Comment:
If you are going to use `clickCount`, then maybe change the counter example as well, because `count` and `clickCount` is confusing.

Reply:
What should I change it to?

Reply:
I don't know, maybe come up with a different example. @Mytrill had a todo list originally, but I thought it was too long, so I replaced it for the current counter.

Reply:
I put it there so we can call this action in the `up` and `down` action to demonstrate that the types are working properly.

Reply:
If it's not a new example, then maybe just refactor the names.

Reply:
IMHO, the test should have an example of asynchronous action (i.e. returning a promise) and an action where the return type is actually used by the view, since these could potentially be shortcomings of these types, which were not there with the previous version.